docs/DeveloperGuide.md

```diff
@@ -985,11 +1059,81 @@ Success action: When a person is successfully assigned:
        Expected (No Alice Tan): No person to unassign. Error details is shown in the status message, as the NAME does not belong to anyone in the address book.
 
 
-
 ### Saving data
 
 1. Dealing with missing/corrupted data files
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
-
-1. _{ more test cases …​ }_
+    1. Prerequisites: Have the app running with some sample data saved
+
+    2. Test case: Delete addressbook.json
+        * Steps:
+            1. Navigate to data folder (default: `[App Home]/data/`)
+            2. Delete `addressbook.json`
+            3. Restart the application
+        * Expected:
+            - App creates new `addressbook.json` with empty data
```

Comment:
i think right now if you delete the data folder or json file it'll create the address book w SampleDataUtil stuff. Only if its corrupted it will create an empty address book.